eng/pipelines/ci.yml

```diff
@@ -111,12 +111,6 @@ jobs:
         condition: and(succeededOrFailed(), eq(variables['PythonVersion'], '3.11'))
         workingDirectory: $(Build.SourcesDirectory)/autorest.python/packages/autorest.python/test/multiapi
 
-      - script: |
```

Comment:
I think it is related with security? @weshaggard

Reply:
Yes this is related to security we can no longer use SAS tokens and instead of fixing this tooling to work with other related authentications we decided to remove these coverage reports.